lib/vast/Interfaces/CMakeLists.txt

```diff
@@ -30,3 +35,23 @@ add_vast_interface_library(DefaultDataLayoutTypeInterface
 add_vast_interface_library(ElementTypeInterface
   ElementTypeInterface.cpp
 )
+
+add_vast_interface_library(ASTContextInterface
+  ASTContextInterface.cpp
+)
+
+add_vast_interface_library(ASTDeclInterface
+  ASTDeclInterface.cpp
+)
+
+add_vast_interface_library(ASTExprInterface
+  ASTExprInterface.cpp
+)
+
+add_vast_interface_library(ASTStmtVisitorInterface
+  ASTStmtVisitorInterface.cpp
+)
+
+add_vast_interface_library(ASTTypeInterface
+  ASTTypeInterface.cpp
+)
```

Comment:
I do not see a reason not to have these in one library?

Reply:
I packed it into one library.